### PR TITLE
Add docs about _scopes option on enum

### PR DESCRIPTION
**Pull request by morgoth**
[ci skip]

### CODE REVIEW

File: activerecord/lib/active_record/enum.rb [hunk 1]
```diff
@@ -35,6 +35,12 @@ module ActiveRecord
   #   Conversation.archived
   #   Conversation.not_archived
   #
+  # Defining scopes can be disabled by setting +:_scopes+ to +false+.
+  #
+  #   class Conversation < ActiveRecord::Base
+  #     enum status: [ :active, :archived ], _scopes: false
+  #   end
+  #
```
Comment (carlosantoniodasilva) [inline activerecord/lib/active_record/enum.rb]: Can you please move this to after the next paragraph/examples? I think it makes the reading flow a little better to show that you can query before showing you how to disable. Make sure to squash / keep a single commit. Thanks!

Comment (morgoth) [reply activerecord/lib/active_record/enum.rb]: @carlosantoniodasilva done

**Comment by carlosantoniodasilva** [comment]
Cherry-picked in 2ef9ecaf454f69db256da01001ad6cb0d5609072. (I added `[ci skip]` to the commit message, forgot to ask you before.) Thanks!

**Comment by morgoth** [comment]
I actually added `[ci skip]` in original commit but it didn't have any effect. I guess it was working only on Travis.

**Pull request closed without merging**